### ISSUE TITLE
sync 01.11.2021

### ISSUE THREAD

**Issue by kuzdogan** [sourcifyeth/org#74]
- npm package initial version out (branch [contract-call-decoder](https://github.com/ethereum/sourcify/tree/contract-call-decoder))
  - Let Kris from [eth-sdk](https://github.com/dethcrypto/eth-sdk) know about it. He'll take a look
- Hardhat will output metadata in the upcoming version by default. We can use only the `build-info` output file for verification as it contains the Source code too.
  - Should we work on it this week?
- Gather down twice
- Second RPC of networks was not getting called. I.e. when gather is down, it is not falling back to Alchemy. Pushed [a fix](https://github.com/ethereum/sourcify/commit/d63a96c3f22a46c2eaa32762160b3ab627f4d476)
- @ligi requested the Sourcify repo to be pinned on Hetzner https://github.com/ethereum/devops/issues/829 Hetzner has unlimited network + we can use https://ipfs.ethdevops.io/ directly as the gateway
- Erigon meeting Wednesday 4pm

** What's next **
- Hardhat support?
- Now that the npm package is out, I want to create a simple UI for decoding the bytecode to metadata IPFS hash. Etherscan contract page has IPFS hash but it is not decoded correctly. Will create a Twitter thread like "Nobody realized Etherscan show incorrect info?". Then people can copy the raw bytecode to get the ipfs hash and a link to it. Goals: promote metadata, Sourcify, and the contract-call-decoder package. 
  - Should I host it under sourcify.dev (e.g. metadata.sourcify.dev) or somewhere else?
- "Verifying Optimism contracts in one go?" @chriseth 
- @ogwurujohnson on "View in Remix" feature

**Comment by franzihei** [comment]
Hey just as a side comment, I think it would really help if we would do bullet point style outcomes / what has been decided in meetings in this issue as well. Otherwise it's very hard to follow if you miss a meeting..
